etc/rucio.cfg.atlas.client.template

```diff
@@ -14,4 +15,6 @@ rucio_host = https://voatlasrucio-server-prod.cern.ch:443
 auth_host = https://voatlasrucio-auth-prod.cern.ch:443
 ca_cert = $RUCIO_HOME/etc/ca.crt
 client_x509_proxy = $X509_USER_PROXY
-request_retries = 3
\ No newline at end of file
+request_retries = 3
+support_experiment = hn-atlas-dist-analysis-help@cern.ch
```

Comment:
An random user can change/edit this directly. Maybe in some policy files (like in permission ?) looks better

Reply:
They can edit the python files as well 😄  Imho the rucio.cfg is the right place for it, as this will be provided by the VO anyway.

Reply:
is that really realistic? ;-)

Reply:
Few things like permission, webui pages, user support are related to a vo/policy. Maybe having all this information handled in the same way is beneficial.

Reply:
That argument makes more sense.

However, we still need the vo-distinction in the rucio.cfg, which we don't have right now (only for permissions).

Reply:
yes good point. Now it's :
[permission]
policy=atlas

can we make it more generic ? like 

[policy]
organyzation=atlas

and have some policies files

Reply:
yeah, i was thinking something like that. also sounds more generic and reasonable to me.